n3fit/src/n3fit/tests/test_evolven3fit.py

```diff
@@ -111,7 +111,7 @@ def test_utils():
             np.testing.assert_allclose(my_PDF.xfxQ2(pid, x, q20), x * (1.0 - x))
     # Testing read_runcard
     runcard = utils.read_runcard(REGRESSION_FOLDER)
-    assert runcard["description"] == "n3fit regression test"
+    assert runcard["description"] == "evolven3fit utils test"
```

Comment:
```suggestion
    assert isinstance(runcard["description"], str)
```

The exact string is probably not important